app/apps/users/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Users(User):
+    phone_num = models.IntegerField()
```

Comment:
It would be the best to accept following phone numbers:
- `+48782211111`
- `+905456311111`

Reply:
So `CharField` might be your choice with some validation like:
- first char is `+`
- country code
- number of digits for particular country

More in: https://docs.djangoproject.com/en/3.2/ref/validators/